mkdocs.yaml

```diff
@@ -59,3 +59,18 @@ markdown_extensions:
 plugins:
     - search
     - markdown-exec
+    - gen-files:
+          scripts:
+              - docs/gen_ref_pages.py
+    - literate-nav:
+          nav_file: SUMMARY.md
+    - section-index
+    - mkdocstrings:
+          handlers:
+              python:
+                  import:
+                      - https://docs.python.org/3/objects.inv
+                      - https://numpy.org/doc/stable/objects.inv
+                      - https://pandas.pydata.org/docs/objects.inv
```

Comment:
I think there's no `pandas` or `numpy` here hopefully? :sweat_smile:

Reply:
No, this is just copy-pasted from another project of mine that does use those libraries. I'll get these unused parts out.